build.gradle

```diff
@@ -64,7 +64,7 @@ compileJava {
     sourceCompatibility = 8
     targetCompatibility = 8
     options.encoding = 'UTF-8'
-    options.compilerArgs = [ '-Xlint:all', '-Werror' ]
+    options.compilerArgs = [ '-Werror', '-Xlint:all', '-Xlint:-deprecation' ]
```

Comment:
It is part of our strategy to ignore _own_ deprecations.

One of the next PRs will remove all `@SuppressWarnings("deprecation")` annotations. This is out of the scope of this PR.